package.json

```diff
@@ -264,6 +270,7 @@
     "react-test-renderer": "16.8.6",
     "svg-tag-names": "^1.1.1",
     "through": "^2.3.8",
+    "typescript": "^4.2.4",
```

Comment:
I think `prettier` is going to need to be upgraded as well. `prettier@1.14.3` is pretty old and crashes on new TypeScript syntax such as `import type ...`.

```
$ C:\Code\emotion\node_modules\.bin\prettier packages\utils\src\index.ts
[error] packages\utils\src\index.ts: SyntaxError: '=' expected. (1:13)
[error] > 1 | import type { RegisteredCache, EmotionCache, SerializedStyles } from './types'
[error]     |             ^
```

Reply:
Could the syntax omit `type` until prettier is upgraded? Or are there other blockers present too.

Reply:
We could just upgrade Prettier - probably best to do it in a separate, dedicated, PR after removing flow types.

Reply:
https://github.com/emotion-js/emotion/pull/2387 for reference.

My two cents: a Prettier upgrade is generally a nice thing and doesn't relate to the rest of the TypeScript stuff. Could it come in separately, then get brought into the `ts-migration` as a merge from master?

Reply:
Merged 🎊